content/courses/solana-pay/solana-pay.md

```diff
@@ -235,11 +235,15 @@ async function post(req: PublicKey, res: PublicKey) {
     feePayer: account,
   });
 
-  const instruction = SystemProgram.transfer({
+    const instruction = SystemProgram.transfer({
     fromPubkey: account,
+    fromPubkey: new PublicKey(account),
```

Comment:
You're adding a second `fromPubkey`. Also the indentation is broken.

Reply:
Thank you for your feedback! I've updated the code to remove the duplicate and fixed the indentation.